sonar-project.properties

```diff
@@ -0,0 +1,13 @@
+sonar.projectKey=gravitee-io_gravitee-management-webui
+sonar.organization=gravitee-io
+
+# Path to sources
+sonar.sources=src
+
+# Source encoding
+sonar.sourceEncoding=UTF-8
+
+# Coverage
+sonar.test=src
+sonar.test.inclusions=**/*.spec.ts
```

Comment:
only spec ? no node_modules ?

Reply:
apparently SC does it alone ok

Reply:
It's not the excluded files but the included tests